strapi/strapi
# feat(content-releases): added a purchase content releases page

### What does it do?

When not on the right plan, you should not be able to access Releases and you should see a lock when you are not on the right plan (Cloud Pro or Community Edition).

When you click on it you should see a page that tells you you don’t have access (just like for Audit Logs, SSO or Review Workflows but in the main menu).

### Why is it needed?

Because we want to give visibility even if the user is not on the right plan and promote the feature

### How to test it?

- open an instance of Strapi without license, in the CE edition
- on the Left Menu you will see the Releases Link with a lock, it links you to the `/purchase-content-releases` page
- inside the promotional page you will see a link to this page https://strapi.io/pricing-self-hosted?utm_campaign=Growth-Experiments&utm_source=In-Product&utm_medium=Releases
